osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/main/java/org/opensmartgridplatform/adapter/protocol/dlms/infra/messaging/requests/to/core/OsgpRequestMessageSender.java

```diff
@@ -27,10 +28,29 @@
   @Qualifier("protocolDlmsOutboundOsgpCoreRequestsJmsTemplate")
   private JmsTemplate jmsTemplate;
 
+  @Autowired
+  @Qualifier("protocolDlmsReplyToQueue")
+  private ActiveMQDestination replyToQueue;
+
   public void send(
       final RequestMessage requestMessage,
       final String messageType,
       final MessageMetadata messageMetadata) {
+    this.send(requestMessage, messageType, messageMetadata, false);
+  }
+
+  public void sendWithReplyToThisInstance(
+      final RequestMessage requestMessage,
+      final String messageType,
+      final MessageMetadata messageMetadata) {
+    this.send(requestMessage, messageType, messageMetadata, true);
+  }
```

Comment:
Why 2 send functions? If the boolean has a good name, it should be clear, right?

Reply:
I try to avoid boolean parameters in public functions, because from the outside it's not immediately clear what the difference is between `send(message, true)` or `send(message, false)`. Of course, a good name for the boolean will help the reader to figure out what the boolean means, but I prefer a meaningful function name,